pom.xml

```diff
@@ -117,6 +118,7 @@
   <developers/>
   <!--TODO-->
   <properties>
+    <hbase.thirdparty.gson.version>2.8.5</hbase.thirdparty.gson.version>
     <maven.javadoc.skip>true</maven.javadoc.skip>
     <maven.build.timestamp.format>
       yyyy-MM-dd'T'HH:mm
```

Comment:
About same as the gson-only module PR but messaging is more muddled here I'd say. I like option #2.